hw3/02-charts.js

```diff
@@ -0,0 +1,105 @@
+const backgroundColors = [
```

Comment:
A lot of your chart is grey, you may want to add more colors to this and the borderColors so each house has a different color

Reply:
I wasn't sure if that was the point because we weren't given more even though there are more characters? I thought it was intentional, like they are lesser characters? oh well